month/months.js

```diff
@@ -0,0 +1,102 @@
+const today = new Date(Date.now())
+let currentMonth = today.getMonth()
+let currentYear = today.getFullYear()
+
+document.querySelector("#months-next").addEventListener('click', () => {
+    incrementMonth(1)
+    fillMonth(currentYear, currentMonth)
+})
+document.querySelector("#months-prev").addEventListener('click', () => {
+    incrementMonth(-1)
+    fillMonth(currentYear, currentMonth)
+})
+
+function updateLabel(){
+    const monthViewLabelMonth = document.querySelector("#monthView-label-month")
+    const monthViewLabelYear = document.querySelector("#monthView-label-year")
+    monthViewLabelMonth.innerHTML = new Date(currentYear, currentMonth).toLocaleDateString('us-US', {month: 'long'})
+    monthViewLabelYear.innerHTML = currentYear
+}
+
+function incrementMonth(value){
+    currentMonth += value
+    if(currentMonth > 11){
+        currentMonth = currentMonth%12
+        currentYear++
+    }
+    if(currentMonth < 0){
+        currentMonth += 12
+        currentYear--
+    }
+}
+
+function putInContainer(element){
+    return `<div class="container monthView-cell">${element}</div>`
+}
+
+function generateCell(content){
+    const element = 
+    `<button 
+        class="button button-round monthView-button"
+        data-currentmonth=${content.currentMonth}
+        data-currentday=${content.currentDay}
+        >
+        ${content.date}
+    </button>`
+    return putInContainer(element)
+}
+
+function fillMonth( year, month ){
+    // const today = new Date(Date.now());
+    const monthView = getMonthArray(year, month);
+    let html = ''
+    monthView.forEach(elem => {
+        html += generateCell(elem)
+    })
+
+    const months = document.querySelector("#month")
+    months.innerHTML = html
+
+    updateLabel()
+}
+
+function getMonthArray(year, month){
```

Comment:
better name would be `getDaysInMonth`
generally, you don't want to put programmatic concepts, such as "Array" in your names, unless "Array" is somehow part of your business/logic domain.
But you do want to describe what exactly the function does